api-reference/10 UI Components/BaseChart/7 Chart Elements/Label/3 Methods/shift(x_y).md

```diff
@@ -3,16 +3,141 @@ id: baseLabelObject.shift(x, y)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Moves label to the specified location.
 
 ##### return: Object
-<!-- Description goes here -->
+The label's instance.
 
 ##### param(x): Number
-<!-- Description goes here -->
+The `x` coordinate.
 
 ##### param(y): Number
-<!-- Description goes here -->
+The `y` coordinate.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The origin is at the upper left corner of the Chart. The coordinates represent the distance between a label and the origin and should be positive numbers.
```

Comment:
All comments are pretty minor by themselves, but together they warrant an edit: 

- We try to substitute the word "represent" with a synonym where possible. "Represent" is a bit vague. 
- "And should be" - creates an additional clause that is in the very end, but refers to the very first noun of the sentence. So I re-arrange it all a little bit. 
- If I measure distance, I would mention "origin" first. Looks like it's natural to start from zeroes. So I'm changing "between label and origin" to "between origin and label". 


```suggestion
The origin is at the upper-left corner of the Chart. The coordinates are positive numbers that specify the distance between the origin and the label.
```